manifests/install.pp

```diff
@@ -1,26 +1,8 @@
 class parse_platform::install {
 
-  exec { 'parseserver_install':
-    command  => 'npm install -g parse-server',
-    user     => 'root',
-    provider => 'shell',
-    unless   => '/usr/bin/test -d /usr/lib/node_modules/parse-server'
-  }
-
-  upstart::job { 'parse-server':
-    description         => 'parse-server',
-    start_on            => 'runlevel [2345]',
-    stop_on             => 'runlevel [016]',
-    respawn             => true,
-    respawn_limit       => '5 10',
-    chdir               => '/tmp',
-    env                 => {
-      'PATH'            => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
-      'APPLICATION_ENV' => $environment
-    },
-    script              => "
-    exec /usr/lib/node_modules/parse-server/bin/parse-server --appId ${parse_platform::application_id} --masterKey ${parse_platform::master_key}
-    ",
-  }
 
```

Comment:
Es importante repetar la guia de estilo. https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace

Recomiendo adaptar tu editor a la sintaxis del lenguaje con el que trabajas.

Tu usas notepad++, asi que ...

- http://mattparkes.net/notepad-as-an-ide-for-puppet/

- https://github.com/opticyclic/npp-puppet-lint